evalml/pipelines/components/transformers/preprocessing/delayed_feature_transformer.py

```diff
@@ -16,10 +16,11 @@ class DelayedFeatureTransformer(Transformer):
     hyperparameter_ranges = {}
     needs_fitting = False
 
-    def __init__(self, max_delay=2, delay_features=True, delay_target=True, gap=1, random_seed=0, **kwargs):
+    def __init__(self, date_index=None, max_delay=2, delay_features=True, delay_target=True, gap=1, random_seed=0, **kwargs):
         """Creates a DelayedFeatureTransformer.
 
         Arguments:
+            date_index (str): Ignored
```

Comment:
nit: Period to match the other descriptions?

Reply:
Sure I can prepend the ignored statement with a description!